drivers/adin2111/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.29)
+
+if(NOT ENABLE_TESTING)
+    set(CMAKE_TOOLCHAIN_FILE ${CMAKE_SOURCE_DIR}/../../cmake/arm-gcc-toolchain.cmake)
+endif()
```

Comment:
Is this necessary if it is defined at the top level?

Reply:
Nope — should change 👍